### DISCUSSION TITLE
I enabled subtitles at the https://updraft.cyfrin.io/ but i cant see them

### DISCUSSION THREAD

**Discussion by nikita-biichuk** [Cyfrin/foundry-full-course-cu#2458] [General]
Where can I find the subtitles?

**Comment by EngrPips** [comment]
Hello @nikitabiichuk2009, The image below shows the icon to click on for subtitles

<img width="1440" alt="Screenshot 2024-09-12 at 18 00 11" src="https://github.com/user-attachments/assets/acd96bbc-0721-4c6a-8bff-183cf728e413">

**Reply by nikita-biichuk** [reply]
Thanks a lot!

**Reply by EngrPips** [reply]
You are welcome, Friend.
